### ISSUE TITLE
"Escalation" workflow to break ties, handle large discrepencies in feedback.

### ISSUE THREAD

**Issue by e0d** [edx/edx-ora2#1952]
ﬂexibility to add graders/staﬀ tosome submissions as necessary(e.g. when we see 2 peer scoresdiﬀer by a huge amount- Ability to edit a single instance of a peer grade of an ora response  (teacher grade overrides one of the peer scores and then recalculates)

**Comment by jswope00** [comment]
Yes, I often see requests for Faculty Override. I think any submission should have the option of faculty override for simplicity sake. 
And maybe a simple way to do this is have a "grade confidence" calculation or something that just highlights submissions with a wide deviation between scores (red for wide deviation. Green for small deviation. yellow in between), and then faculty can look at those and determine if they need to intervene.
